noxfile.py

```diff
@@ -78,10 +80,12 @@ def test_with_datadog_as_required_package(session: nox.Session):
 def test_with_xray_sdk_as_required_package(session: nox.Session):
     """Tests that depends on AWS XRAY SDK library"""
     # Tracer
+    # Middleware factory with tracer
     build_and_run_test(
         session,
         folders=[
-            f"{PREFIX_TESTS_FUNCTIONAL}/tracing/aws_xray_sdk/",
+            f"{PREFIX_TESTS_FUNCTIONAL}/tracing/_aws_xray_sdk/",
+            f"{PREFIX_TESTS_FUNCTIONAL}/middleware_factory/_aws_xray_sdk/",
```

Comment:
I had to add `_` before the name of the folder to avoid conflict with the XRay SDK name.

Reply:
maybe use `/tracer` instead?

does that work?

Reply:
Yeah, sure! Check now pls.